source/classes.tex

```diff
@@ -1866,7 +1866,7 @@
 assignment operator is defined as deleted; otherwise, it is defined as
 defaulted\iref{dcl.fct.def}.
 The latter case is deprecated if the class has a user-declared copy constructor
-or a user-declared destructor.
+or a user-declared destructor  \iref{depr.impldec}.
```

Comment:
Two spaces?

Reply:
@tkoeppe I believe that cross reference is already there in [expr.prim.lambda.capture]p7, just before the example.  I did a search on the word 'deprecated' and these were the only two uses without a cross reference.

@BitonicEelis thanks, good catch.  Force-pushed the quick correction to avoid cluttering the history when merged.

Reply:
Oh, very good, thanks!